packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -20,6 +20,19 @@ const ignoredContracts = [
   // These contracts are not in the Registry (before release 1)
   'ReserveSpenderMultiSig',
   'GovernanceApproverMultiSig',
+
+  // These are Mento contracts which we are no longer maintaining
+  'Exchange',
```

Comment:
Does this work if we try to check an old release? In some parts of the code we add the ignore only after a certain version, Does that make sense here?

Reply:
I updated the code to ignore those contracts only for version 9 and higher